### PR TITLE
准备脚本功能

### PR DESCRIPTION
**Pull request by lliioollcn**

### CODE REVIEW

**Comment by lliioollcn** [comment]
其他的看别人吧hhhh
或者以后我慢慢写

File: app/src/main/java/nil/nadph/qnotified/script/QNScript.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    private final Interpreter instance;
+    private boolean enable;
+
+    public QNScript(Interpreter lp) {
+        this.instance = lp;
+    }
+
+    public void enable() {
+        try {
+            instance.eval("enable()");
```
Comment (cinit) [inline app/src/main/java/nil/nadph/qnotified/script/QNScript.java:19]: 建议用onEnable()之类的

Comment (lliioollcn) [reply app/src/main/java/nil/nadph/qnotified/script/QNScript.java:19]: ok，下次提交改一下

File: app/lint.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <issue id="InvalidPackage" severity="ignore"></issue>
```
Comment (cinit) [inline app/lint.xml:3]: Please use this:

<?xml version="1.0" encoding="UTF-8"?>
<lint>
    
    
    <issue id="InvalidPackage">
        <ignore regexp='^.*java.applet..*$'/>
        <ignore regexp='^.*java.awt..*$'/>
        <ignore regexp='^.*javax.script..*$'/>
        <ignore regexp='^.*javax.servlet..*$'/>
        <ignore regexp='^.*javax.swing..*$'/>
    </issue>
</lint>

**Comment by cinit** [comment]
改一下lint, gkd

**Pull request merged**